node/src/components/event_stream_server/event.rs

```diff
@@ -19,6 +24,11 @@ pub enum Event {
         block_hash: BlockHash,
         execution_result: Box<ExecutionResult>,
     },
+    Equivocation {
```

Comment:
it's still _equivocation_ . Please change to `Fault`.

Reply:
I have one of the events named as `Evidence` on the SSE side, so I shall change it `Evidence`?

Reply:
Renamed to `Fault`